tests/jest/components/AdminSettings.spec.js

```diff
@@ -1183,9 +1196,35 @@ describe('AdminSettings.vue', () => {
 				})
 			})
 
-			describe('user_oidc app disabled', () => {
+			describe('Unsupported user_oidc app enabled', () => {
+				beforeEach(async () => {
+					wrapper = getWrapper({ state: { ...state, ...authorizationSettingsState, user_oidc_enabled: true, user_oidc_supported: false } })
+				})
+				it('should show field values and hide authorization settings form', () => {
+					const authorizationSettingsForm = wrapper.find(selectors.authorizationSettings)
+					expect(wrapper.vm.isIntegrationCompleteWithOIDC).toBe(true)
+					expect(authorizationSettingsForm.element).toMatchSnapshot()
+				})
+				it('should disable reset button', () => {
+					const resetButton = wrapper.find(selectors.authorizationSettingsResetButton)
+					expect(resetButton.attributes().disabled).toBe(undefined)
```

Comment:
`undefined` means not disabled
```suggestion
					expect(resetButton.attributes().disabled).toBe('true')
```